src/mlpack/methods/ann/rbm/binary_rbm_policy.hpp

```diff
@@ -0,0 +1,177 @@
+/**
+ * @file binary_rbm_policy.hpp
+ * @author Kris Singh
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_RBM_BINARY_RBM_POLICY_HPP
+#define MLPACK_METHODS_ANN_RBM_BINARY_RBM_POLICY_HPP
+
+#include <mlpack/core.hpp>
+#include <mlpack/prereqs.hpp>
+#include <mlpack/methods/ann/activation_functions/softplus_function.hpp>
+#include <mlpack/methods/ann/activation_functions/logistic_function.hpp>
+
+namespace mlpack{
+namespace ann{
+template <typename DataType = arma::mat>
```

Comment:
I think we should add a line between the namespace and the class template to improve the readability, what do you think?